waltz-ng/client/common/services/icon_names.js

```diff
@@ -32,7 +32,7 @@ export const entityIconNames = {
     CAPABILITY: 'puzzle-piece',
     CHANGE_INITIATIVE: 'paper-plane-o',
     DATABASE: 'database',
-    DATA_TYPE: 'wpforms',
+    DATA_TYPE: 'qrcode',
```

Comment:
I didn't like the last one either.

On Thu, Aug 11, 2016 at 5:22 PM, David Watkins <notifications@github.com>
wrote:

> In waltz-ng/client/common/services/icon_names.js
> <https://github.com/khartec/waltz/pull/417#discussion_r74454885>:
>
> > @@ -32,7 +32,7 @@ export const entityIconNames = {
> >      CAPABILITY: 'puzzle-piece',
> >      CHANGE_INITIATIVE: 'paper-plane-o',
> >      DATABASE: 'database',
> > -    DATA_TYPE: 'wpforms',
> > +    DATA_TYPE: 'qrcode',
>
> like this better than the last one!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/khartec/waltz/pull/417/files/e4bc887bdadc42f03d5e44e0876776ce30ea1a5c#r74454885>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC452_m6i36RSPP4YKUryWdL1ovICAglks5qe0xLgaJpZM4JiBOI>
> .
>



-- 


*Kamran Saleem*
t  +44 (0) 7732 078654
e  kamran@thinkincode.co.uk
w thinkincode.co.uk